Cargo.toml

```diff
@@ -9,14 +17,9 @@ members = [
 	"system-parachains/bridge-hubs/bridge-hub-kusama",
 	"system-parachains/bridge-hubs/bridge-hub-polkadot",
 	"system-parachains/collectives/collectives-polkadot",
+	"system-parachains/gluttons/glutton-kusama",
 ]
 
-[workspace.package]
-authors = ["Parity Technologies <admin@parity.io>"]
-edition = "2021"
-repository = "https://github.com/polkadot-fellows/runtimes.git"
-version = "1.0.0"
```

Comment:
aren't all runtimes going to be published at the same time? what is the use-case for separate versioning?

is it going to be publish to crates.io at all? what is needed for?

Reply:
Hm, yea good question. Maybe we can just keep the workspace version, @bkchr ?

Reply:
I see zero point of following semvar here. Runtime isn't a library. So yeah let's keep visioning simple.

Reply:
Changed to use the workspace now.